.cirrus.yml

```diff
@@ -5,12 +5,13 @@ environment:
 timeout_in: 3h
 
 windows_docker_builder:
+  only_if: $CIRRUS_TAG != ''
   use_compute_credits: true
   platform: windows
   os_version: 2019
   info_script: docker info
   pull_script: docker pull mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019
-  build_core_script: docker build -m 2GB --tag cirrusci/windowsservercore:2019 windowsservercore
```

Comment:
Seems like we still need to build these images for backward compatibility 🤔

Reply:
Yeah, it still builds. I just added another tag. So the old image will always point to the latest version.